app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsViewModel.kt

```diff
@@ -49,6 +50,9 @@ class SessionsViewModel @Inject constructor(
     val isLoading: LiveData<Boolean> by lazy {
         tab.map { it.inProgress }
     }
+    val viewPool: RecyclerView.RecycledViewPool by lazy {
```

Comment:
Can you add the comment here?

Reply:
Like this
// Share RecyclerViewPool between view pager content fragments

Reply:
Added comment on a1f9bbf !